torch/distributed/elastic/multiprocessing/api.py

```diff
@@ -388,6 +389,7 @@ def __init__(
         tee_stdouts: Dict[int, str],
         tee_stderrs: Dict[int, str],
         error_files: Dict[int, str],
+        log_line_headers: Dict[int, str],
```

Comment:
nit: can we make these `log_line_prefix`? I know I used `header` in `tail_logfile()` but in various documentations we refer to this feature as "localrank log prefixing"